boards/arc/nsim/haps_arcv3_init.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 Synopsys
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <zephyr/init.h>
+
+#define ARC_CLN_MST_NOC_0_0_ADDR	292
+#define ARC_CLN_MST_NOC_0_0_SIZE	293
+
+#define ARC_CLN_MST_NOC_0_1_ADDR	2560
+#define ARC_CLN_MST_NOC_0_1_SIZE	2561
+
+#define ARC_CLN_MST_NOC_0_2_ADDR	2562
+#define ARC_CLN_MST_NOC_0_2_SIZE	2563
+
+#define ARC_CLN_MST_NOC_0_3_ADDR	2564
+#define ARC_CLN_MST_NOC_0_3_SIZE	2565
+
+#define ARC_CLN_MST_NOC_0_4_ADDR	2566
+#define ARC_CLN_MST_NOC_0_4_SIZE	2567
+
+#define ARC_CONNECT_CMD_CLN_PER0_BASE	2688
+#define ARC_CONNECT_CMD_CLN_PER0_SIZE	2689
+
+#define AUX_CLN_ADDR			0x640
```

Comment:
Why mixing hex and decimal in this list? I'd prefer hex for all.

Reply:
It's different address spaces (AUX address space and cluster address space which is accessible via two AUX regs) - so even in documentation they are defined differently (AUX regs in hex, cluster regs in decimal).

Though I can recalculate everything to hex, if you prefer so.

Reply:
if this is consistent with the documentation, then let's leave it like this.